### ISSUE TITLE
Typescript Cannot add any headers to a fetch api

### ISSUE THREAD

**Issue by asmanp2012** [AliMD/vatr#73]
### Package(s)

fetch

### Description

The fetch header type has an error.

### Please provide the exception or error you saw (Preferably with a screenshot)

```bash
Types of property 'headers' are incompatible.
    Type '{ 'Content-Type': string; }' is not assignable to type 'Headers | string[][]'.
```

### Device or platform

Device: Desktop or Phone
OS: windows
Browser: Chrome
Version: 0

### Vatr registered list

```console
[
  {
    "name": "@vatr/fetch",
    "vertion": "1.2.3"
  }
]
```

**Comment by AliMD** [comment]
Currect way to solve

```ts
options.headers = {
      ...options.headers,
      'Content-Type': 'application/json',
    };
```

**Comment by MM25Zamanian** [comment]
What is fix commit message ??
and What is add me to contributors ?? @AliMD

**Comment by AliMD** [comment]
`fix(fetch): build issue #73`

`chore(packgae): update contributors`
